mission/functions/tasks/defend/fn_task_defend_counterattack.sqf

```diff
@@ -66,10 +66,12 @@ _taskDataStore setVariable ["INIT", {
 	private _areaSize = markerSize _marker;
 
 	// search for candidate FOBs within the zone's area.
-	private _base_search_area = [_markerPos, _areaSize select 0, _areaSize select 1, 0, false];
+	private _base_search_area = [_markerPos, vn_mf_bn_s_zone_radius + 100, vn_mf_bn_s_zone_radius + 100, 0, false];
```

Comment:
`_areaSize` relates to the SGD on map marker. but FOBs can be built outside that area as we've extended the zone size.